src/coreclr/jit/llvm.cpp

```diff
@@ -2392,6 +2458,48 @@ void Llvm::PlaceAndConvertShadowStackLocals()
     lowerToShadowStack();
 }
 
+static bool placeLocalInAlloca(LclVarDsc* varDsc)
+{
+    return !varDsc->lvInSsa;
+}
+
+void Llvm::createAllocasForLocalsWithAddrOp()
+{
+    m_allocas = new std::vector<Value*>(_compiler->lvaCount, nullptr);
+
+    for (unsigned lclNum = 0; lclNum < _compiler->lvaCount; lclNum++)
+    {
+        LclVarDsc* varDsc = _compiler->lvaGetDesc(lclNum);
+
+        // TODO-LLVM LCLBLK - outgoing arg space
```

Comment:
Is there a performance or size penalty in doing that?

Reply:
As far as the Jit is concerned, it can be thought to be "better" because it allows "nested" calls in arguments, thus fewer temps.

`FEATURE_FIXED_OUT_ARGS` is really about the low-level ABI detail of whether the stack pointer remains fixed throughout the method, or can change (via `push` instructions on x86). We don't need to worry about that when targeting LLVM (except for the shadow stack but there we decide ourselves and do not concern the rest of the Jit with those decisions).